pkg/ccl/backupccl/schedule_pts_chaining.go

```diff
@@ -89,10 +130,21 @@ func maybeUpdateSchedulePTSRecord(
 			return nil
 		}
 
-		// If SchedulePTSChainingRecord was not set during backup planning, we do
-		// not need to perform any chaining.
+		// Update the full schedule to disable chaining since there is no associated
+		// inc schedule. It could have been dropped before we got here.
+		if args.BackupType == ScheduledBackupExecutionArgs_FULL && args.DependentScheduleID == 0 {
```

Comment:
huh, we don't do full-to-full chaining if there is no inc schedule?

Reply:
we don't today nope, but you've got me thinking... Today, a full backup with revision history will only backup all revisions since the GC threshold. Do we want to change this to, since the last full backup in the absence of incremental backups?

Reply:
no, makes sense. I'm not even sure full should allow rev history at all.